{{cookiecutter.github_project_name}}/setup.py

```diff
@@ -33,13 +33,15 @@
 ]
 
 data_files_spec = [
-    ('share/jupyter/nbextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/static', '*.*'),
+    ('share/jupyter/nbextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/nbextension', '*.*'),
```

Comment:
Wondering if something like this could filter the `.map` files out:

```suggestion
    ('share/jupyter/nbextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/nbextension', '"**/*[!.map]"'),
```

Haven't checked locally for this cookiecutter, but used in: https://github.com/voila-dashboards/voila/blob/50577c32718e2c15c84a74b9986663b0e241f9a1/setup.py#L56

Reply:
I just updated the webpack config to not generate map files in production mode.

I don't think that glob will do what you want. Globs have very limited syntax for excluding things.

Reply:
I think `[!.map]` matches anything except `.`, `m`, `a`, `p`, as individual characters.

Reply:
See https://docs.python.org/3/library/fnmatch.html#module-fnmatch

Reply:
OK thanks for sharing.

I was under the impression `jupyter-packaging` was transforming the glob into a regex, and it seems to do the job at least in the case of voila linked above.

Reply:
1. I stand corrected about it naively using globs - it does appear to do a lot of work translating globs to regexes.
2. There is an explicitly `exclude` parameter that serves this purpose nicely too: https://github.com/jupyter/jupyter-packaging/blob/master/jupyter_packaging/setupbase.py#L151

Reply:
Looks like that glob translates to what I was saying above:

```
In [3]: jupyter_packaging.setupbase._translate_glob('**/*[!.map]')
Out[3]: '(?ms).*[^/]*[^.map]\\Z'
```

Reply:
For example:

```
In [19]: m = re.compile(jupyter_packaging.setupbase._translate_glob('**/*[!.map]'))

In [20]: bool(m.match('test.p'))
Out[20]: False
```

Reply:
Actually, it seems `**/*[!.map]` means "include any file, except if the file ends in a `.`, `m`, `a`, or `p`". For example:
```
In [10]: bool(m.match('testp'))
Out[10]: False
```

Reply:
> Actually, it seems **/*[!.map] means "include any file, except if the file ends in a ., m, a, or p". For example:

Ah that would explain why it matches `.map` files then (since they end with a `p`). Thanks for the details :+1: